### DISCUSSION TITLE
github pro student discount code!

### DISCUSSION THREAD

**Discussion by sl194** [community/community#61181] [GitHub Education]
### Hello everyone,

does anyone happen to know where to reapply for a new academic discount coupon for github pro account?

**Comment by samus-aran** [comment]
Hi @sl194 👋 

You can re-verify your GitHub Global Campus student status either by:

- receiving an email stating your student status needs to be re-verified. This should include a link to the application page to re-verify.
- return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

Hope this helps 👍
